test/unit/test_images.py

```diff
@@ -4,11 +4,11 @@
 import pytest
 from harvey.globals import Global
 from harvey.images import Image
-
+from docker.models.images import ImageCollection
 
 @pytest.mark.parametrize('context', [('test'), (None)])
-@mock.patch('subprocess.check_output')
-def test_build_image(mock_subprocess, context, mock_webhook):
+@mock.patch.object(ImageCollection, 'build')
+def test_build_image(mock_build, context, mock_webhook):
     # TODO: Mock the subprocess better to ensure it does what it's supposed to
```

Comment:
Let's kill this comment since we aren't using the subprocess anymore.

Reply:
Ok. I'll make this change